cmake/configs/default.cmake

```diff
@@ -99,6 +99,7 @@ hunter_config(convertutf VERSION 1.0.1)
 hunter_config(crashpad VERSION v0.0.1-p0)
 hunter_config(crashup VERSION 0.0.2)
 hunter_config(cvmatio VERSION 1.0.27-p3)
+hunter_config(czmq VERSION 4.0.1-p0)
```

Comment:
tl;dr just follow the [rules](https://github.com/ingenue/hunter) blindly and everything will be ok :)

the path of changes is `hunter/develop` -> `ingenue/pkg.template` -> `ingenue/pkg.*` -> `ingenue/upload.*`. All "regular" changes should go to `hunter/develop`, all package-specific CI configuration (i.e. `.travis.yml`/`appveyor.yml`) should go to `ingenue/pkg.*`. I will take care of the rest. For now [pkg.czmq](https://travis-ci.org/ingenue/hunter/builds/180869545) branch contains only of CI configuration without real example so it has "red" state, hence it's okay that your pull request will not make it "green" (you're not breaking working stuff).